tests/sentry_plugins/twilio/test_plugin.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import absolute_import
+
+from exam import fixture
+from sentry.testutils import TestCase, PluginTestCase
+from sentry_plugins.twilio.plugin import TwilioConfigurationForm, TwilioPlugin
+
+
+class TwilioConfigurationFormTest(TestCase):
+    def test_valid_form(self):
+        form = TwilioConfigurationForm(
+            data={
+                "sms_from": "3305093095",
+                "sms_to": "330-509-3095, (330)-509-3095, +13305093095",
+                "auth_token": "foo",
+                "account_sid": "bar",
+            }
+        )
+
+        self.assertTrue(form.is_valid())
+        self.assertDictEqual(
+            form.clean(),
+            {
+                "auth_token": u"foo",
+                "sms_to": u"+13305093095,+13305093095,+13305093095",
```

Comment:
@mattrobenolt Is this a feature or a bug? I'd have expected `sms_to` to not have redundant numbers.

Reply:
I guess that's fair. It probably should de-dupe, and this test was just testing normalization.

Reply:
Oh, the actual plugin code which doesn't have coverage right now appears to use `split_sms_to` for dedupe. Just gonna tack on dedupe to `clean_sms_to` to be safe. e726dcba79de305e2d008c2c6ec0c99b1834e304